Centaurus.Models/Effects/Effect.cs

```diff
@@ -25,7 +25,9 @@ public abstract class Effect
         //ignore it during the serialization - we need it only to decide which effects to send back to a particular user
         public AccountWrapper AccountWrapper { get; set; }
 
-        [XdrField(0)]
         public long Apex { get; set; }
+
+        [XdrField(0)]
+        public int Account { get; set; }
```

Comment:
Why do you need `Account` here? We discussed that effects should always be transferred with quantum (in message result or history requests), so there is no need in the account ref field. That's 4 bytes but we serialize a huge amount of effects.

Reply:
We do need this property (we discussed it). One quantum can affect multiple accounts. So we need to know what account was affected by an effect. I know how we can drop it, but we need to discuss it.